llvm/lib/Transforms/PeepholeOptimizationCourse/ReplaceInt8AddNewPassManager/ReplaceInt8SumNewPassManager.cpp

```diff
@@ -0,0 +1,31 @@
+#include "llvm/Transforms/PeepholeOptimizationCourse/ReplaceInt8SumNewPassManager.h"
```

Comment:
Did you dupliate sources for passes in include and Transforms folders?

Reply:
I’m sorry, it wasn’t done intentionally, I’ll fix it with the new commit